scripts/claude-slide/verify_slide_mechanical.py

```
@@ -0,0 +1,815 @@
+
+def resolve_manual_dir(arg: str) -> str:
+    arg = arg.rstrip("/")
+    if os.path.isabs(arg):
+        return arg
+    if os.path.isdir(arg):
+        return os.path.abspath(arg)
+    return os.path.join(PROJECT_ROOT, arg)
+
+
+def find_source_html(manual_dir: str) -> str:
+    """元 HTML（slide_* / verify_* 以外の .html）を 1 つ返す。"""
```

Ruff (0.15.17)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)